app/views/schools/dashboard/_school_cohort_details.html.erb

```diff
@@ -37,12 +37,20 @@
   <% if school_cohort.full_induction_programme? %>
     <% list.row do |row| %>
       <% row.key(text: "Training provider") %>
-      <% row.value(text: lead_provider&.name ? lead_provider.name : "To be confirmed") %>
+      <% if latest_partnership&.challenged? == true %>
```

Comment:
```suggestion
      <% if latest_partnership&.challenged? %>
```

Do you need the condition here (and below)?

Reply:
Both are required because there are different behaviours for true, false and nil values.

`false`: the LP and DP names, when it's not challenged
`true`: no LP and DP names, when it's challenged
`nil`: "To be confirmed", when the partnership does not exist yet (pending)